product_images_downloader.py

```diff
@@ -23,39 +22,37 @@ def extract_product_divs(html):
     return product_divs
 
 
+def extract_images(product_divs):
 
-def extract_images(product_divs) :
-    
     # just a wrapper to iterate each product and extract img-link
-    
 
-    for i in range(len(product_divs)) :   # for each product
-        uid = product_divs[i].get('data-asin')    # for any product, this unique id will be present...
-        try :
+    for i in range(len(product_divs)):   # for each product
+        # for any product, this unique id will be present...
+        uid = product_divs[i].get('data-asin')
+        try:
+
+            prod = product_divs[i]
 
-            prod = product_divs[i]  
-            
             img_link = prod.find('img').get('src')
 
             download_img(img_link, uid)   # download img and saves the image
 
-        except :
+        except:
             print(uid)   # products for which image is failed to download
             continue
-            
-    
-def download_img(img_url, uid) :
+
+
+def download_img(img_url, uid):
 
     # it will store elements in the images folder. make sure it already exists !!!
     img = requests.get(img_url).content
-    
-    with open(f'images/{uid}.jpg', 'wb') as f :
+
+    with open(f'output/{uid}.jpg', 'wb') as f:

```

Comment:
I was initially confused by your output-1.png folder/file. I initially assumed that was where the generated files went, and did not know I lacked the correct initial folder to make downloads. I've renamed to what I would have expected from the outset.